gcsio/src/main/java/com/google/cloud/hadoop/gcsio/InvocationIdGrpcChannelInterceptor.java

```diff
@@ -0,0 +1,57 @@
+package com.google.cloud.hadoop.gcsio;
+
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.ForwardingClientCall.SimpleForwardingClientCall;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import java.util.UUID;
+
+public final class InvocationIdGrpcChannelInterceptor implements ClientInterceptor {
+
+  static final Metadata.Key<String> GOOG_API_CLIENT =
+      Metadata.Key.of("x-goog-api-client", Metadata.ASCII_STRING_MARSHALLER);
+
+  static final String GCCL_INVOCATION_ID = "gccl-invocation-id";
```

Comment:
what is gccl?

Reply:
rename this to GCCL_INVOCATION_ID_PREFIX since this is not really the invocationId

Reply:
Updated.